stm32/README.md

```diff
@@ -0,0 +1,61 @@
+# Building for the stm32
+
+## Quickstart
+
+1. Build codec2 for your host system, see codec2/README.md and codec2/INSTALL
+
+2. Install agcc-arm-none-eabi toolchain:
```

Comment:
We should mention NOT to use the distributions packages as they are often outdate (or even have serious issues such as the one in Ubuntu 18.04).